.github/workflows/publish-technical-documentation-release.yml

```diff
@@ -0,0 +1,29 @@
+name: publish-technical-documentation-release
+
+on:
+  push:
+    branches:
+      - release-*
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    paths:
+      - 'docs/sources/tempo/**'
+  workflow_dispatch:
+jobs:
+  sync:
+    if: github.repository == 'grafana/tempo'
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: grafana/writers-toolkit/publish-technical-documentation-release@publish-technical-documentation-release/v1
```

Comment:
Just for my understanding, it looks like there are separate actions for next and versioned updates.

`publish-technical-documentation` takes the destination folder entirely `content/docs/tempo/next`
`publish-technical-documentation-release` takes the parent folder `content/docs/tempo` and adds the version internally.

Reply:
You are exactly right both about the two separate actions and the way they behave. I may consolidate them later but I can do that internally.